app/controllers/sessions_controller.rb

```diff
@@ -7,6 +7,12 @@ def create
     if @user&.authenticate(params.dig(:session, :password))
       reset_session
       log_in @user
+      if params.dig(:session,
```

Comment:
dùng toán tử 3 ngôi nhé e vì nó ngắn để viết trên 1 dòng 
 params.dig(:session, :remember_me) == "1" ?  remember(@user) :  forget(@user)

Reply:
vaang a